sdk/appplatform/Azure.ResourceManager.AppPlatform/samples/Generated/Samples/Sample_AppPlatformDeploymentResource.cs

```diff
@@ -477,7 +477,7 @@ public async Task GenerateThreadDump_DeploymentsGenerateHeapDump()
         public async Task StartJfr_DeploymentsStartJFR()
         {
             // Generated from example definition: specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2022-12-01/examples/Deployments_StartJFR.json
-            // this example is just showing the usage of "Deployments_StartJfr" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Deployments_StartJFR" operation, for the dependent resources, they will have to be created separately.
```

Comment:
`JFR` is what is defined in swagger: https://github.com/Azure/azure-rest-api-specs/blob/c852a76b96e5c71972cdc62a303665fec7c867b6/specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2023-12-01/appplatform.json#L6271